packages/react-form/src/hooks/listdirty.ts

```diff
@@ -0,0 +1,5 @@
+import {ListBag} from '../types';
+
+export function useListDirty(lists?: ListBag) {
+  return lists ? lists.dynamicList.dirty : false;
+}
```

Comment:
```suggestion
export function useListDirty(lists: ListBag) {
  return lists.dynamicList.dirty;
}
```
I think we can assume lists is defined here otherwise there is no need to seek if it's dirty.
But this hook seems a bit overkilled no? do we really need it?

Reply:
Ah I guess this would be useful if we allow several dynamic lists. We would have to check each dirty state.

Reply:
In that case I will do `lists && lists.dynamicList.dirty` and keep as optional param